infra/src/main/kotlin/fi/fta/geoviite/infra/util/FreeText.kt

```diff
@@ -30,7 +32,9 @@ data class FreeTextWithNewLines private constructor(private val value: String) :
     Comparable<FreeTextWithNewLines>, CharSequence by value {
 
     companion object {
-        const val ALLOWED_CHARACTERS = FreeText.ALLOWED_CHARACTERS + "\n"
+        const val ALLOWED_CHARACTERS = FreeText.ALLOWED_CHARACTERS + NEW_LINE_CHARACTER
+        const val ESCAPED_NEW_LINE = "\\n"
```

Comment:
Jos näitä on tarkoitus näyttää loppukäyttäjille niin `\n`-merkkijonoksi escapettamisen sijasta korvaisin nämä tyhjillä merkkijonoilla. En usko että meidän loppukäyttäjät on niin tietoisia tietokoneiden merkkijonoesitysten sisäisestä toiminnasta että he tietäisivät ainakaan järjestäen mitä `\n` edustaa. Ja silloin kun itsekin näen niitä missään koodieditorissa lukevan koodarin määrittelemän stringin ulkopuolella, niin oletan kyseessä on bugi eikä ominaisuus.

Reply:
Ehkä tästä voisi turista vielä dailyn yhteydessä tms., mutta en itse näe mahdollisuutta muodostaa täsmällinen alkuperäinen viesti jollain käänteisoperaatiolla kovinkaan olennaisena loppukäyttäjille (veikkaan että CSV:tä oikeasti lukeville ihmisille se aiheuttaisi lähinnä sekaannusta jos/kun tulee vastaan outoja merkkejä/merkkijonoja,) ja siksi joko trimmaisin ne edelleen kokonaan pois tai korkeintaan korvaisin whitespacella.